### PR TITLE
Added proper field types to consumer info to match upstream kafka

### PR DESCRIPTION
**Pull request by serejja**

### CODE REVIEW

File: structs.go [hunk 1]
```diff
@@ -71,14 +71,14 @@ func (a byId) Less(i, j int) bool { return a[i].Id < a[j].Id }
...
 //General information about Kafka consumer. Used to keep it in consumer coordinator.
 type ConsumerInfo struct {
-	Version      int16
-	Subscription map[string]int
-	Pattern      string
-	Timestamp    int64
+	Version      int16          `json:"version"`
+	Subscription map[string]int `json:"subscription"`
+	Pattern      string         `json:"pattern"`
+	Timestamp    string         `json:"timestamp"`
```
Comment (miry) [inline structs.go]: @serejja you can use `Timestamp int64 `json:",string"``

Comment (serejja) [inline structs.go]: @miry cool, please see updated PR

**Comment by miry** [comment]
@serejja It is good

But it would not  support  `{"Version": "1", "Timestamp": "12313123", "Subscription": {"test": "1"}}`. The Subscriptions would be empty.

I will check later who put to zookeeper `string` instead of `int`.

**Comment by serejja** [comment]
@miry yes please, do so, because it feels to me like this should be fixed in the library that puts that kind of data and not should we be able to handle all cases

**Comment by serejja** [comment]
@miry any updates? can we merge this now or maybe there's something else that should be added? thanks

**Comment by serejja** [comment]
@miry I'm merging this now, let me know if we need some additional changes to this

**Comment by miry** [comment]
@serejja it is all ok. thanks.

**Pull request merged**